allennlp_server/commands/configure.py

```diff
@@ -2,13 +2,14 @@
 The ``configure`` subcommand launches a webapp that helps you generate an AllenNLP configuration file.
 
     $ allennlp configure --help
-    usage: allennlp configure [--help] [-h HOST] [-p PORT] [--include-package INCLUDE_PACKAGE]
+    usage: allennlp configure [-h] [--host HOST] [-p PORT]
+                              [--include-package INCLUDE_PACKAGE]
 
     Run a config explorer for allennlp.
 
     optional arguments:
-      --help                show this help message and exit
-      -h HOST, --host HOST  interface to serve the wizard on (default = 127.0.0.1)
+      -h, --help            show this help message and exit
+      --host HOST           interface to serve the wizard on (default = 127.0.0.1)
```

Comment:
`argparse` doesn't allow me to override it, it throws an exception. Don't know how `flask run` does it, but it doesn't matter...